test/test_format.py

```diff
@@ -375,7 +375,7 @@ def test_makeboxes_outputform_text(
         ("Subsuperscript[a, p, q]", "a_p^q", None),
         (
             "Integrate[F[x],{x,a,g[b]}]",
-            "\\int_a^{g\\left[b\\right]} F\\left[x\\right] \uf74cx",
+            "\\int_a^{g\\left[b\\right]} F\\left[x\\right] \\, dx",
```

Comment:
I found this by chance: In LaTeX, the integral measure should be put with a 'd' instead the unicode character.
For some reason, the changes in this PR fixed it.

Reply:
Ok. Put in another PR just for  that or just change it on mater.